packages/modal/tests/Modal.test.tsx

```diff
@@ -13,44 +14,44 @@ describe("Modal", () => {
   describe("ModalBase", () => {
     it("renders ModalBase", () => {
       const action = jest.fn();
-      const component = mount(
+      const { asFragment } = render(
         <ModalBase isOpen={true} onClose={action}>
           <div tabIndex={0}>I am modal content</div>
         </ModalBase>
       );
-      expect(toJson(component)).toMatchSnapshot();
+      expect(asFragment()).toMatchSnapshot();
     });
-    it("calls this.props.onClose when keying 'esc'", () => {
+    it("calls this.props.onClose when keying 'esc'", async () => {
       const action = jest.fn();
-      const component = mount(
+      const { getByRole } = render(
         <ModalBase isOpen={true} onClose={action}>
           <div tabIndex={0}>I am modal content</div>
         </ModalBase>
       );
+      const dialog = getByRole("dialog");
 
       expect(action).not.toHaveBeenCalled();
-      component.find('[role="dialog"]').simulate("keyDown", {
-        key: "Escape"
-      });
+      await userEvent.type(dialog, "[Escape]");
       expect(action).toHaveBeenCalled();
     });
     it("calls this.props.onClose when clicking dark overlay", () => {
       const action = jest.fn();
-      const component = mount(
+      const { getByRole } = render(
         <ModalBase isOpen={true} onClose={action}>
           <div tabIndex={0}>I am modal content</div>
         </ModalBase>
       );
+      const closeButton = getByRole("button");
 
       expect(action).not.toHaveBeenCalled();
-      component.find('[role="button"]').first().simulate("click");
+      fireEvent.click(closeButton);
```

Comment:
did you want to try `userEvent.click` here?

Reply:
Not really. I did mix and match in this test, I wanted userEvent to use a key press. I've generally been using fireEvent.click() elsewhere.

Reply:
Unless you think it's worth doing for testing purposes? Happy to update it.